package.json

```diff
@@ -81,13 +82,6 @@
     "type": "git",
     "url": "https://github.com/aws-amplify/amplify-cli.git"
   },
-  "husky": {
-    "hooks": {
-      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS",
-      "pre-push": "yarn build-tests-changed && yarn split-e2e-tests",
```

Comment:
we shouldn't drop these hooks.

Reply:
Oh....I just ran `npx mrm@2 lint-staged` ([code](https://prettier.io/docs/en/precommit.html#option-1-lint-stagedhttpsgithubcomokonetlint-staged)) and didn't pay attention to the change 😂

Reply:
Good catch. Added them back 293ce82c0824dc4117c26fd96be8857f0352a238